pkg/app/piped/platformprovider/lambda/client.go

```diff
@@ -287,6 +272,10 @@ func (c *client) updateFunctionConfiguration(ctx context.Context, fm FunctionMan
 			Environment: &types.Environment{
 				Variables: fm.Spec.Environments,
 			},
+			VpcConfig: &types.VpcConfig{
+				SecurityGroupIds: fm.Spec.VPCConfig.SecurityGroupIDs,
+				SubnetIds:        fm.Spec.VPCConfig.SubnetIDs,
+			},
```

Comment:
Should check nullity for `fm.Spec.VPCConfig` before call to its fields

Reply:
I fixed it.